src/fortran_routing/mc_pylink_v00/MC_singleSeg_singleTS/courant_dev/v02_routing_test_instructions.md

```diff
@@ -0,0 +1,54 @@
+# Test v02 with new edits to M-C routing module
+## What changed?
+- At the core of this pull request are changes to `MCsingleSegStime_f2py_NOLOOP.f90`. Now this `muskingcungenwm` subroutine in this module returns three diagnostic variables: kinematic celerity, courant number, and M-C X parameter. 
+- To accommodate this change with v02 routing, I edited `pyMCsingleSegStime_NoLoop.f90`, to accept three additional output variables from `muskingcungenwm`, while only returning qdc, velc, and depthc. As such, the `mc_reach.compute_network` and specifically `mc_reach.compute_reach_kernel` will run just fine. However, while additional diagnostic variables provided by `muskingcungenwm`, they are not yet "seen" or made available to the `mc_reach`.
+- To allow testing of v02, I also edited `compute_nhd_routing_SingleSeg_v02.py` to add the `/fast_reach` directory to the system path, else it is unable to find and load `mc_reach`.
+
+## How to test that Adam did not break v02
+### 0. Clone this branch, or figure out how to work in Google Colab.
+### 1. Compile v02 cython
+- While the compile commands will be unique to your system requirements, here is what works with A. Wlostowski's system (a shell script):
+
+```
+# define directory path variables, assumed root = /t-route
+FORTRAN_ROUTING_DIR="/src/fortran_routing/mc_pylink_v00/MC_singleSeg_singleTS"
+FORTRAN_RES_DIR="/src/fortran_routing/mc_pylink_v00/Reservoir_singleTS"
+FAST_REACH_DIR="/src/python_routing_v02/fast_reach"
+
+cd $FORTRAN_ROUTING_DIR
+gfortran varPrecision.f90 -c -O0 -fPIC
+gfortran -c -O0 -fPIC -o mc_single_seg.o MCsingleSegStime_f2py_NOLOOP.f90
+gfortran pyMCsingleSegStime_NoLoop.f90 -c -o pymc_single_seg.o -O3 -fPIC
+cp $FORTRAN_ROUTING_DIR/*.o $FAST_REACH_DIR 
+
+cd $FORTRAN_RES_DIR
+gfortran varPrecision.f90 -c -O0 -fPIC
+gfortran -c -O2 -fPIC -o module_levelpool.o module_levelpool.f90
+gfortran -c -O2 -fPIC -o pymodule_levelpool.o pymodule_levelpool.f90
+cp $FORTRAN_RES_DIR/*.o $FAST_REACH_DIR 
+
+# define more directory path variables - system specific!
+numpy_I="/home/awlostowski/tr/lib/python3.8/site-packages/numpy/core/include"
+py_I="/usr/include/python3.8"
+py_lib="/home/awlostowski/tr/lib"
```

Comment:
We don't want these kinds of home paths even just in instructions. A generic `/home/user/venv/lib` might be better.

Reply:
roger that. changed,